core/cmd/node_commands_integration_test.go

```diff
@@ -0,0 +1,141 @@
+//go:build integration
+
+package cmd_test
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+
+	"github.com/pelletier/go-toml/v2"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/config"
+	"github.com/smartcontractkit/chainlink/v2/core/cmd"
+	"github.com/smartcontractkit/chainlink/v2/core/internal/cltest"
+	"github.com/smartcontractkit/chainlink/v2/core/internal/testutils/cosmostest"
+	"github.com/smartcontractkit/chainlink/v2/core/services/chainlink"
+)
+
+func cosmosStartNewApplication(t *testing.T, cfgs ...chainlink.RawConfig) *cltest.TestApplication {
+	//for i := range cfgs {
+	//	cfgs[i].SetDefaults() //TODO?
+	//}
```

Comment:
we can drop these now since defaults are applied on the plugin side, but we still need to manage the common ones, like `Enabled`